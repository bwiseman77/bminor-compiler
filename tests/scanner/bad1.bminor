/* "this is bad?\n
/* "but this isnt?
