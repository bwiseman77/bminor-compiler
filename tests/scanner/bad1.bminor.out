-------------------------------------------
           token type  text
-------------------------------------------
             DIVISON  /
      MULTIPLICATION  *
          SCAN ERROR  "
               IDENT  this
               IDENT  is
               IDENT  bad
        TERNARY_PT_1  ?
          SCAN ERROR  \
               IDENT  n
             DIVISON  /
      MULTIPLICATION  *
          SCAN ERROR  "
               IDENT  but
               IDENT  this
               IDENT  isnt
        TERNARY_PT_1  ?
